### PR TITLE
Basic changes to get Jenkins working

### PR DESCRIPTION
**Pull request by techgnosis**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -85,68 +81,53 @@ spec:
...
           sh "gcloud config set compute/zone ${env.CLUSTER_ZONE}"
           sh "gcloud config set core/project ${env.PROJECT_ID}"
-          sh "gcloud config set container/cluster ${env.CLUSTER_NAME}"
           sh "gcloud config set compute/region ${env.REGION}"
          }
         }
     }
 
-    stage('linter') {
+    stage('configure environment file') {
```
Comment (chrislovecnm) [inline Jenkinsfile]: All of this file needs code level docs

File: blue-green-upgrade/validate.sh [hunk 2]
```diff
@@ -118,10 +116,10 @@ validate_control() {
 # 0 - when all node versions == NEW_K8S_VER
 # 1 - when a node version != NEW_K8S_VER
 validate_nodes() {
-  NODES=$(kubectl get nodes -o name)
+  NODES=$(kubectl -n default get nodes -o name)
   for NODE in ${NODES}; do
     # Find the kubelet version on each node.  This will match the gke version
-    NODE_VER=$(kubectl get "${NODE}" -o json | \
+    NODE_VER=$(kubectl -n default get "${NODE}" -o json | \
```
Comment (chrislovecnm) [inline blue-green-upgrade/validate.sh]: can you just set the current context?  Not sure why we are setting the default namespace every time

File: test/configure-jenkins-environment.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (chrislovecnm) [inline test/configure-jenkins-environment.sh]: need to use env on this please

File: Jenkinsfile (hunk 1 above)
Comment (techgnosis) [reply Jenkinsfile]: Comments have been added to the configure environment script

File: blue-green-upgrade/validate.sh (hunk 2 above)
Comment (techgnosis) [reply blue-green-upgrade/validate.sh]: I get what you're saying but so far the precedent has been to set in other projects to set namespace explicitly. We can look at doing it a different way in a different PR.

**Pull request merged**